helmfile/charts/opensearch/configurer/values.yaml

```
@@ -3,6 +3,7 @@ image: elastisys/curl-jq:latest
 
 dashboard:
   ck8sVersion: ""
+  isElastisysCustomer: ""
```

You seem to have missed adding this to the values file in the grafana chart.

```suggestion
dashboard:
  ck8sVersion: ""
  isElastisysCustomer: false
```

---

This isn't necessary for the grafana chart, they are a bit different.

Not sure if this has to be set because it will read the value from `sc-config.yaml` if set or the default value in `defaults/sc-config.yaml` which is already set to false.

---

Yeah it's not strictly needed there. But we still try to include all possible options in those values files. So I would like you to add it there.